test-suite/shared/src/test/require-jdk11/testsuite/javalib/lang/StringTestOnJDK11.scala

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
The file path is missing `org/scalajs/`:
```diff
-test-suite/shared/src/test/require-jdk11/testsuite/javalib/lang/StringTestOnJDK11.scala
+test-suite/shared/src/test/require-jdk11/org/scalajs/testsuite/javalib/lang/StringTestOnJDK11.scala
```
On master there is already `CharacterTestOnJDK11.scala` in the `test-suite/shared/src/test/require-jdk11/org/scalajs/testsuite/javalib/lang/` directory, so `StringTestOnJDK11.scala` should be next to it.

Reply:
Fixed and squashed commits.